db/seeds.rb

```diff
@@ -474,28 +505,75 @@
 view_count = 1000
 start_date = DateTime.parse("2015-09-20 00:00:00")
 end_date = DateTime.parse("2015-10-20 00:00:00")
+created_at = '20-09-2015'.to_datetime.in_time_zone("Singapore") - 8.hours
+user_id = 1000
+while user_id < 1078
+  limit = user_id + rand(4..12)
+  while user_id < limit
+    Viewcount.create(id: view_count, deal_id: 1000, user_id: user_id, created_at: created_at, entry: 'merchant_push_notification')
+    view_count = view_count + 1
+    user_id = user_id + 1
+    created_at = created_at + rand(1..3).hours
+  end
+end
+user_id = 1000
 while start_date <= end_date
   limit = view_count + rand(20..40)
   while view_count < limit
-    Viewcount.create(id: view_count, deal_id: 1000, user_id: view_count, created_at: start_date)
+    if user_id == 1078
```

Comment:
@Woonyong92 What's e purpose of setting this if condition to reset user_id to 1000?